### PR TITLE
[openhab] Add new chart.

### PR DESCRIPTION
**Pull request by reitermarkus**
**Description of the change**

Add chart for OpenHAB.

**Benefits**

N/A

**Possible drawbacks**

N/A

**Applicable issues**

N/A

**Additional information**

**Checklist** 
- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [ ] Variables are documented in the README.md (this can be done with using our helm-docs wrapper `./hack/gen-helm-docs.sh stable <chart>`)

### CODE REVIEW

File: charts/incubator/openhab/values.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        enabled: true
+        port: 8443
+
+ingress:
+  # -- Enable and configure ingress settings for the chart under this key.
+  # @default -- See values.yaml
+  main:
+    enabled: false
+
+# -- Configure persistence settings for the chart under this key.
```
Comment (bjw-s) [inline charts/incubator/openhab/values.yaml:46]: I would put a remark here indicating that the user should select one of the two approaches.

**Review by bjw-s — CHANGES_REQUESTED** [review]
Thanks for taking the time to make this chart and submitting the PR! 🍻 

The change looks good in general, just a small textual requested to provide some clarification to the users. Once that's updated and the README regenerated I'll approve merge it :)

**Comment by reitermarkus** [comment]
Added clarification for volume options.

**Review by onedr0p — APPROVED** [review]

**Comment by onedr0p** [comment]
@all-contributors please add @reitermarkus for code

**Pull request merged**